proto/kvrpcpb.proto

```diff
@@ -1151,3 +1151,44 @@ message RawChecksumResponse {
     uint64 total_kvs = 4;
     uint64 total_bytes = 5;
 }
+
+message CompactError {
+    oneof error {
+        CompactErrorInvalidStartKey err_invalid_start_key = 1;
+        CompactErrorPhysicalTableNotExist err_physical_table_not_exist = 2;
+        CompactErrorCompactInProgress err_compact_in_progress = 3;
+        CompactErrorTooManyPendingTasks err_too_many_pending_tasks = 4;
+    }
+}
+
+message CompactErrorInvalidStartKey {}
+
+message CompactErrorPhysicalTableNotExist {}
+
+message CompactErrorCompactInProgress {}
+
+message CompactErrorTooManyPendingTasks {}
+
+message CompactRequest {
+    // If specified, the compaction will be performed starting from this start key.
```

Comment:
Add comments about the behavior when the start_key is not set

Reply:
Thanks for the suggestion! More comments have been added to important fields.